staging/src/k8s.io/api/resource/v1beta1/types.go

```diff
@@ -436,10 +436,11 @@ type DeviceRequest struct {
 	//   count field.
 	//
 	// - All: This request is for all of the matching devices in a pool.
+	//   At least one device must be available on the node for the allocation to succeed.
```

Comment:
/lgtm cancel

Actually, let's avoid "available" in the API docs :sweat_smile: That may have triggered this whole discussion thread.

```suggestion
	//   At least one device must exist on the node for the allocation to succeed.
```

Reply:
done

Reply:
/lgtm